src/_includes/graphql/item-selected-bundle-option.md

```diff
@@ -0,0 +1,19 @@
+#### ItemSelectedBundleOption attributes {#ItemSelectedBundleOption}
+
+The ItemSelectedBundleOption object contains a list of bundle options that are assigned to the bundle product.
+
+Attribute | Data type | Description
+--- | --- | ---
+`id` | ID! | The unique identifier of the option
+`label` | String! | The label of the option
+`values` | [[ItemSelectedBundleOptionValue!](#ItemSelectedBundleOptionValue)]! | A list of products that represent the values of the parent option
```

Comment:
`ItemSelectedBundleOptionValue!` is not mandatory in the code. I don't see any reason why it should not be mandatory and is probably a bug. But for documenting lets reflect what we are shipping. I will log a P3 for this.

Reply:
@prabhuram93 The docs are wrong. The object was probably required at some point in 2.4.1.